documentation.html

```diff
@@ -43,7 +43,111 @@ <h1>Components</h1>
+                </p>
+                <div class="avatar-size-desc">
+                    <p><span class="avatar-size-info">avatar-xxl</span> :12rem</p>
+                    <p><span class="avatar-size-info">avatar-xl</span> :10rem</p>
+                    <p><span class="avatar-size-info">avatar-l</span> :8rem</p>
+                    <p><span class="avatar-size-info">avatar-m</span> :6rem</p>
+                    <p><span class="avatar-size-info">avatar-s</span> :4rem</p>
+                </div>
+                <h5>Examples of Round Image Avatar:</h5>
+                <div class="container">
+                    <div class="avatar">
+                        <img class="ava-round-img avatar-xxl" src="./assets/avatar.png" alt="Avatar" /></div>
+                    <div class="avatar">
+                        <img class="ava-round-img avatar-xl" src="./assets/avatar.png" alt="Avatar" /></div>
+                    <div class="avatar">
+                        <img class="ava-round-img avatar-l" src="./assets/avatar.png" alt="Avatar" /></div>
+                    <div class="avatar">
+                        <img class="ava-round-img avatar-m" src="./assets/avatar.png" alt="Avatar" /></div>
+                    <div class="avatar">
+                        <img class="ava-round-img avatar-s" src="./assets/avatar.png" alt="Avatar" /></div>
+                </div>
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28133%2C122%2C53%2C1%29&t=night-owl&wt=sharp&l=auto&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=152%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522ava-round-img%2520avatar-xxl%2522%2520src%253D%2522%2522%2520alt%253D%2522Avatar%2522%2520%252F%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522ava-round-img%2520avatar-xl%2522%2520src%253D%2522%2522%2520alt%253D%2522Avatar%2522%2520%252F%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522ava-round-img%2520avatar-l%2522%2520src%253D%2522%2522%2520alt%253D%2522Avatar%2522%2520%252F%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522ava-round-img%2520avatar-m%2522%2520src%253D%2522%2522%2520alt%253D%2522Avatar%2522%2520%252F%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522ava-round-img%2520avatar-s%2522%2520src%253D%2522%2522%2520alt%253D%2522Avatar%2522%2520%252F%253E%250A%253C%252Fdiv%253E"
+                    style="width: 590px; height: 393px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
```

Comment:
The inline styling in iframe could be removed

Reply:
Thanks for bringing it to my notice, I had forgotten about this.
Had made the required changes.